src/codeflare_sdk/demo-notebooks/guided-demos/2_basic_interactive.ipynb

```diff
@@ -65,10 +65,10 @@
     "    head_gpus=1, # For GPU enabled workloads set the head_gpus and num_gpus\n",
     "    num_gpus=1,\n",
     "    num_workers=2,\n",
-    "    min_cpus=2,\n",
-    "    max_cpus=2,\n",
-    "    min_memory=8,\n",
-    "    max_memory=8,\n",
+    "    min_cpus='250m',\n",
```

Comment:
Are these changes necessary?

Reply:
they are needed to make the notebook running in GitHub runner, which has limited CPU and memory